src/PUGizmo/PUGizmo.class.st

```diff
@@ -190,7 +196,7 @@ PUGizmo class >> printInheritanceWithClass: aClass withStream: aStream [
 				<< '"';
 				<< superClass mooseName;
 				<< '"'.
-			superClass isInterface
+			((self checkSmalltalkTypeContainerOf: superClass) not and: [superClass isInterface])
```

Comment:
I think you could do something like:

```st
((superClass class selectors includes: #isInterface) and: [superClass isInterface])
```

Reply:
If you agree to do it this way, I don't mind, but I thought it might have a problem with error reporting.
For example, if a language that is supposed to understand *isInterface* doesn't, it won't cause an error. It may be too heavy but I would have seen a list of languages (*superClass*) allowed to not understand *isInterface*?

```st
((self mustIncludeIsInterface: superClass) and: [superClass isInterface])
```

That said, if you prefer your proposal no worries for me.

Reply:
> For example, if a language that is supposed to understand _isInterface_ doesn't, it won't cause an error.

If I understand your point, I think it's out of the scope of PlantUMLPharoGizmo to check these kinds of errors. That would seem to be a fundamental flaw in a model that should be tested by other software.

I looked at the Midas software for querying Metamodels and it's my understanding that #isInterface is not really a modeling query (you can't query Interfaces, because they're just classes with #isInterface set to true). The models don't distinguish them at the entity level.

Reply:
I agree, in this context, I understand better your choice. Thanks for the explanation.

I close the PR in consequence :)

Reply:
@enwiro Thanks for finding and suggesting the improvement - in fact I made the change and added some more tests in another PR and your issue was the "trigger" for this. 👏🏼

Reply:
Thank you, I hope I can help you again in the future.